packages/fondue/src/components/RichTextEditor/Plugins/ColumnBreakPlugin/types.ts

```diff
@@ -0,0 +1,12 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type PluginProps } from '@components/RichTextEditor/Plugins/Plugin';
+
+export type BreakAfterPluginProps = PluginProps & {
+    columns?: number;
+    gap?: string | number;
```

Comment:
What about having a type like this?

```tsx
import type tailwindConfig from '<root>/tailwind.config';

type Breakpoint = keyof typeof tailwindConfig.theme.screens;
export type Responsive<TValue> = { [key in Breakpoint]?: TValue } | TValue;

export type BreakAfterPluginProps = PluginProps & {
    columns?: Responsive<number>;
    gap?: Responsive<string | number>;
```

Reply:
What we aim to have now is container queries, that are responsive based on the container, not the traditional viewport. For that we will [add the container query plugin to brand-sdk](https://github.com/Frontify/brand-sdk/pull/949) as that is considered block-related logic.